orthw

```diff
@@ -21,6 +21,7 @@ orth="${ort_home}/helper-cli/build/install/orth/bin/orth"
 # Dot directory files:
 #
 dot_dir=".orthw"
+advisor_result_file="$dot_dir/advisor-result.json"
```

Comment:
Adding another file introduces case distinctions to `evaluate()`. 
I also gave it a thought on how advisor relates to the existing steps: analyze, scan, evaluate, report.

`orthw` so far re-used the results of the rather slow data gathering steps, analyze and scan, and re-computes only the fast evaluate and report steps to "re-apply configuration". Using that idea for integrating the advise command, I'd say that the user needs to do any of `orth init advisor-result.json` or `orth init scan-result.json`. 

The only need for any advise command I see is when the advisor result is not up to date. 
Assuming the user intialized with an older ´advisor-result.json`, I'd say adding a "re-advise" command, which drops all existing advises from the initilaized file and then re-fetches advises amending the existing file would be better. This would work without adding any case distinctons or introducing new files.

What do you think ?

Reply:
I get your train of thought and using `orth init advisor-result.json` is not a bad idea the issue I have with that is:

1. We would need to refactor `orthw` to expect  a scan-result.json and advisor-result.json. If the later is passed there is no guarantee the scanner has run which several of `orthw` commands would fail
2. Security advisories can be published at any time and I thought about using `advise` in the `orthw` command but decided against it because there are multiple types of advise providers in ORT. I wanted an easy to remember command so ended up `check-advisories` but I would also be open to calling it `advise-security`. Why I called check-advisories` is what I want to support it a "What are the currently known vulnerabilities" workflow.

Reply:
Not sure if I get this right, so:

1. I believe it's ok to assume that the ORT file passed to `orthw init` contains a scan result. IMO this was assumed so far anyway. So, why can't we continue to assume that a given `advisor-result.json` contains a scan result?
2. My proposal was to have a command (let's disreagard the name it gets) which runs the advisor and replaces any
    advisor result in the initialized ORT file with the advises obtained. So, I believe this would implement the use case "What are the currently known vulnerabilities". What am I missing?